common/common.go

```diff
@@ -91,9 +91,15 @@ const (
 
 //CRD kind
 const (
-	CRDConfig  = "Config"
-	CRDPlugin  = "Plugin"
-	CRDCluster = "Cluster"
+	Config                 = "Config"
+	Plugin                 = "Plugin"
+	Cluster                = "Cluster"
+	KubeConfig             = "KubeConfig"
+	InitConfiguration      = "InitConfiguration"
```

Comment:
I think other modules don't care about those consts, put it into runtime/common.go